### DISCUSSION TITLE
Falha - Item não pode ser encontrado, parâmetro não foi passado corretamente.

### DISCUSSION THREAD

**Discussion by julianoninoka** [RamonSilva20/mapos#1827] [General]
não estou conseguindo criar uma nova, aparece esse erro no modal de alerta, tbm não estou conseguindo add cliente, estou mudando de hospedagem, fiz o backup do sql e importei na nova , porem agora não consigo mais usar o sistema.

**Comment by Pr3d4dor** [comment]
@julianoninoka Qual é o erro que acontece?

**Reply by julianoaraujonk** [reply]
consegui resolver , tinha criado uns campos novos nas tabelas, ai com a nova atualização ficou incompatível e tinha esquecido que eu tinha alterado meu bd
